source/Organism.h

```diff
@@ -149,6 +149,27 @@ class Organism {
     std::cout << "GetLysogeny called from Organism" << std::endl;
     throw "Organism method called!";}
 
+  //Public goods game host functions 
+  virtual void SetPool() {
+    std::cout << "Set resources pool from Organism" << std::endl;
```

Comment:
These should each be the exact name of the method to make it easier to track down if/when it happens